src/main/webapp/assets/modules/technical-detail/technical-detail.con.js

```diff
@@ -1,3 +1,17 @@
-techlooper.controller("technicalDetailController", function() {
+techlooper.controller("technicalDetailController", function (utils, connectionFactory, $routeParams,
+                                                             technicalDetailService, $scope, $timeout, jsonValue) {
+  var term = $routeParams.term;
 
+  // TODO: write a blog about dom manipulation with angularjs
+  $scope.showCircle = function(skill) {
+    technicalDetailService.showSkillsList(skill);
+    return true;
+  }
+
+  connectionFactory.termStatisticInOneYear({term: term})
+    .success(function (data, status, headers, config) {
```

Comment:
'config' is defined but never used.<br>'headers' is defined but never used.<br>'status' is defined but never used.